dist/IO/t/io_linenum.t

```diff
@@ -7,7 +7,7 @@ my $File;
 
 BEGIN {
     $File = __FILE__;
-    require strict; import strict;
+    require strict; strict->import();
```

Comment:
Why not just use strict?

Reply:
I try to avoiding refactoring these tests and preserve the original spirit. I agree a `use strict` will work there.